Set/Game/GameView.swift

```diff
@@ -84,6 +136,14 @@ struct GameView: View {
             .cornerRadius(Constants.buttonCornerRadius)
     }
 
+    private func dealAnimation(for card: CardModel) -> Animation {
+        var delay = 0.0
+        if let index = gameViewModel.deck.firstIndex(where: { $0.id == card.id }) {
+            delay = Double(index) * 2 / Double(12)
+        }
+        return Animation.easeInOut(duration: 0.5).delay(delay)
```

Comment:
Duration of animation can be also moved to the `Constants`, when `Double(index) * 2 / Double(12)` is not required because this is really kind of hardcoded shit 😈

Reply:
fixed